lib/auth.js

```diff
@@ -0,0 +1,25 @@
+let accessToken = process.env.ACCESS_TOKEN
+
+async function auth (ctx, next) {
+  // for health check skipping / and retunrning 200
+  if (ctx.request.url === '/') {
+    return await next()
+  }
+
+  // Check if access_token is present in header
+  if (Object.prototype.hasOwnProperty.call(ctx.request.headers, 'access_token') &&
```

Comment:
yeah actually that is true, in my first approach i also had access_token being passed in as request parameter.